bin/_test-helpers.sh

```diff
@@ -40,8 +40,23 @@ Available Commands:
     --images: by default load images into the cluster from the local docker cache (docker), or from tar files located under the 'image-archives' directory (archive), or completely skip image loading (skip)."
 }
 
+cleanup_usage() {
+  progname="${0##*/}"
+  echo "Cleanup Linkerd integration tests.
+
+Usage:
+    ${progname} [--context k8s_context] [--linkerd_path /path/to/linkerd]
```

Comment:
Nit: If I'm not mistaking we use dashes as word separators in most script parameters, instead of underscores

Reply:
Right! I'll change this

Reply:
This is no longer a flag.